core/src/test/java/org/elasticsearch/index/mapper/IpFieldMapperTests.java

```diff
@@ -186,7 +186,8 @@ public void testIgnoreMalformed() throws Exception {
 
     public void testIncludeInAll() throws Exception {
         String mapping = XContentFactory.jsonBuilder().startObject().startObject("type")
-                .startObject("properties").startObject("field").field("type", "ip").endObject().endObject()
+                .startObject("properties").startObject("field").field("type", "ip")
+                .field("include_in_all", true).endObject().endObject()
```

Comment:
That's the blurb directly below this, previously it checked that it was included in `_all` by default first (without setting the setting), and then that it wasn't when the setting was false; now it checks that it's included in `_all` when explicitly set and is not included when unset entirely.